python/cudf/cudf/pandas/__init__.py

```diff
@@ -25,14 +25,16 @@ def install():
     global LOADED
     LOADED = loader is not None
 
-    if (rmm_mode := os.getenv("CUDF_PANDAS_RMM_MODE", None)) is not None:
+    rmm_mode = os.getenv("CUDF_PANDAS_RMM_MODE", "managed_pool")
+    if rmm_mode != "disable":
```

Comment:
What does "disable" do? Does it just not change the default mr at all? Does that have any meaningful difference from "cuda"? If not, can we remove "disable" as an option and just have "cuda" be the normal default behavior of using cudaMalloc (the default rmm mr)?

Reply:
Dropped `disable` and `cuda` will now act as disabling option.